src/components/BShuttle/BShuttle.test.js

```diff
@@ -0,0 +1,141 @@
+import { shallow } from 'vue-test-utils'
```

Comment:
Does shallow give you enough detail about the child elements, or would mount be better.

Reply:
there are now child components being rendered, so mount isn't needed